joomla/joomla-cms
# [4.0] Add a filter featured to com_contact

### Summary of Changes
Presently, it is not possible to filter for unfeatured contacts in com_contact.
This PR adds a filter "featured" to com_contact


### Testing Instructions
Ensure that you have at least one featured and at least one unfeatured contacts.
Apply the Patch and the list of filters in the Filter Options has a new filter "-Select Featured-".
Apply the filter to your contacts


### Documentation Changes Required
Yes 
New Filter
